app/src/main/java/com/cmput301f20t21/bookfriends/ui/library/OwnedListFragment.java

```diff
@@ -86,6 +91,20 @@ public void onViewCreated(@NonNull View view, @Nullable Bundle savedInstanceStat
         adapter = new OwnedListAdapter(vm.getBooks().getValue(), this::onItemClick, this::onDeleteBook, this::onViewRequests);
         recyclerView.setAdapter(adapter);
 
+        recyclerView.addOnScrollListener(new RecyclerView.OnScrollListener() {
+            @Override
+            public void onScrolled(@NonNull RecyclerView recyclerView, int dx, int dy) {
+                super.onScrolled(recyclerView, dx, dy);
+                if (dy > 0) {
+                    filterButton.hide();
+                    addBookButton.hide();
+                } else {
+                    filterButton.show();
+                    addBookButton.show();
+                }
+            }
```

Comment:
There are few designs that we can use here about the two FABs
1. On scroll down we hide the FABs, on scroll up show the FABs 
    * This is what I currently have and this is what LinkedIn is using in its "My Network" navigation
    * we can remove the bottom padding with this approach 
2. When Scrolling hide the FABs, when stop scrolling show the FABs
<details><summary>Looks like this:</summary>
<p>
![screencast-nimbus-capture-2020 11 06-23_53_09](https://user-images.githubusercontent.com/43459469/98434275-8511a080-208b-11eb-8da3-902cc8de3c86.gif)
</p>
</details>
3. Leave it like before and add more padding to the bottom